doc/source/ray-core/package-ref.rst

```diff
@@ -332,3 +340,11 @@ The Ray Command Line API
 .. click:: ray.scripts.scripts:enable_usage_stats
    :prog: ray enable-usage-stats
    :show-nested:
+
+Job
+~~~
+CLIs to submit, stop, and monitor Ray jobs. See :ref:`Ray Job Submission API Reference <ray-job-submission-api-ref-under-construction>`.
```

Comment:
Q: Should we add the actual reference here, or link should be sufficient?

Reply:
hmm we should consolidate it here (not sure why its all over the place?)

Reply:
Hmm not sure why. For state APIs, I just followed job API (also these 2 features have lots of APIs, and I think it is a bit weird to spread them under core API page).